AWSiOSSDKv2.xcodeproj/project.pbxproj

```diff
@@ -11585,7 +11585,7 @@
 				CLANG_WARN_OBJC_LITERAL_CONVERSION = YES;
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
 				CLANG_WARN_RANGE_LOOP_ANALYSIS = YES;
-				CLANG_WARN_STRICT_PROTOTYPES = NO;
+				CLANG_WARN_STRICT_PROTOTYPES = YES;
```

Comment:
We should change this property to YES only for AWSCore build settings and not the entire project since other modules have some warnings which still need to be resolved.

Reply:
@rohandubal Added some changes to fix the rest of the warnings. Apologies for missing, must have run the wrong target.

Leaving this setting on will hopefully prevent warnings like this in the future